### PR TITLE
Fixes #194: Added search query validation for emojiHeatmapper app

### LINKED ISSUE #194: Add search query validation for emojiHeatmapper app
### Short description
Now the search is also accepting any text in the emojiHeatmapper app. So instead add validation to check check whether the search query is valid emoticon or not.

### PR DESCRIPTION
**Pull request by kavithaenair**
### Short description
Fixes #194 
In this PR, it checks whether the query entered is valid or not. If the query entered is not an emoji, then a alert message pops up.

Demo link: https://kavithaenair.github.io/emojiHeatmapper/  **(Load Scripts)**

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by kavithaenair** [comment]
@djmgit @SKrPl @Achint08 @daminisatya @mariobehling  Please review

File: emojiHeatmapper/script.js [hunk 1]
```diff
@@ -53,6 +53,27 @@ function emptyVector() {
...
+    $.getJSON( "emoji.json", function(tweets) {
+        var flag = 0;
+        for(var i = 0; i < tweets.data.length; i++) {
+            if (tweets.data[i].indexOf(query) !== -1) {
+                flag = 1;
+                break;
+            }
+        }
+        if (flag == 1) {
+             console.log("emoji found plotting map");
```
Comment (Achint08) [inline emojiHeatmapper/script.js]: Remove console messages.

File: emojiHeatmapper/script.js [hunk 2]
```diff
@@ -53,6 +53,27 @@ function emptyVector() {
...
+             updateMap();
+         }
+         else {
+             console.log("emoji not found");
```
Comment (Achint08) [inline emojiHeatmapper/script.js]: Here too.

File: emojiHeatmapper/emoji.json [hunk 3]
```diff
@@ -0,0 +1,1795 @@
...
+    ["emoji","🎢","ROLLER COASTER","amusement park | coaster | roller"],
+    ["emoji","💈","BARBER POLE","barber | haircut | pole"],
+    ["emoji","🎪","CIRCUS TENT","circus | tent"],
+    ["emoji","🎭","PERFORMING ARTS","art | mask | performing | theater | theatre"],
+    ["text*","🖼","FRAME WITH PICTURE","art | frame | museum | painting | picture"],
+    ["emoji","🎨","ARTIST PALETTE","art | museum | painting | palette"],
+    ["emoji","🎰","SLOT MACHINE","game | slot"],
+    ["emoji","🚂","STEAM LOCOMOTIVE\n≊ locomotive","engine | railway | steam | train | vehicle"],
+    ["emoji","🚃","RAILWAY CAR","car | electric | railway | train | tram | trolleybus | vehicle"],
+    ["emoji","🚄","HIGH-SPEED TRAIN","railway | shinkansen |
```
Comment (Achint08) [inline emojiHeatmapper/emoji.json]: Kindly add New line at the end of file

**Comment by kavithaenair** [comment]
@Achint08 Made the changes. Please review.

**Pull request merged**